composer.json

```diff
@@ -12,6 +12,11 @@
     "config": {
         "platform": {
             "php": "8.0.99"
+        },
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "infection/extension-installer": true,
+            "composer/package-versions-deprecated": true
```

Comment:
We should depend on `ocramius/package-versions:^2`, and not on `composer/package-versions-deprecated`, which still uses the problematic approach of `ocramius/package-versions:^1` (long gone)